src/fastcontext/asm.S

```diff
@@ -851,7 +861,11 @@ GET:
 
 .globl SET
 .global SET
-.type SET, %function
+#ifndef ISAPPLEMACHOASM
+ .type GET, %function
```

Comment:
Seems like this changed what was a SET to a GET?

Reply:
That does seem fishy.  @janciesko ?

Reply:
Thanks for reporting. This is wrong. Amended the commit.